servers/cromwell/README.md

```diff
@@ -4,11 +4,7 @@ Thin shim around [`cromwell`](https://github.com/broadinstitute/cromwell).
 
 ## Development
 
-1. Set the `CROMWELL_URL` environment variable to specify which cromwell instance to use. Job Manager pulls workflow data from `https://cromwell.mint-dev.broadinstitute.org/api/workflows/v1` by default:
-
-    ```
-    export CROMWELL_URL=https://cromwell.test-cromwell.broadinstitute.org/api/workflows/v1
-    ```
+1. Set the `CROMWELL_URL` environment variable to specify which cromwell instance to use.
```

Comment:
I think it's still useful to have an example url here

Reply:
Sure, will add it back.